arkane/ess/terachemTest.py

```diff
@@ -399,15 +399,15 @@ def test_load_scan_energies(self):
                            2.30963595e+03, 5.02046166e+03, 7.97285489e+03, 1.06923710e+04,
                            1.29244615e+04, 1.43422341e+04, 1.43905580e+04, 1.32047110e+04,
                            1.12088126e+04, 8.31162367e+03, 5.06568695e+03, 2.54966151e+03,
-                           8.50076205e+02, 0.00000000e+00, 3.31469351e+00]
-        expected_angles = [0., 0.13659098, 0.27318197, 0.40977295, 0.54636394, 0.68295492,
-                           0.81954591, 0.95613689, 1.09272788, 1.22931886, 1.36590985, 1.50250083,
-                           1.63909182, 1.7756828, 1.91227379, 2.04886477, 2.18545576, 2.32204674,
-                           2.45863773, 2.59522871, 2.7318197, 2.86841068, 3.00500167, 3.14159265,
-                           3.27818364, 3.41477462, 3.55136561, 3.68795659, 3.82454758, 3.96113856,
-                           4.09772955, 4.23432053, 4.37091152, 4.5075025, 4.64409349, 4.78068447,
-                           4.91727546, 5.05386644, 5.19045743, 5.32704841, 5.4636394, 5.60023038,
-                           5.73682137, 5.87341235, 6.01000334, 6.14659432, 6.28318531]  # radians
+                           8.50076205e+02, 0.00000000e+00]
+        expected_angles = [0., 0.13962634, 0.27925268, 0.41887902, 0.55850536, 0.6981317,
```

Comment:
Why did these values change?

Reply:
Apparently the values were of an intermediate version of the code, since this test didn't actually run I didn't see that it needed to be updated. These are the corrected expected values (46 entries rather than 47).